test/test-image.rb

```diff
@@ -18,7 +18,7 @@ def setup
         "rdf:about" => "http://www.kuro5hin.org/favicon.ico",
         "#{@prefix}:size" => "small",
       }
-      @favicon_contents = {"dc:title" => "Kuro5hin",}
+      @favicon_contents = {"dc:title" => "Kuro5hin", }
```

Comment:
```suggestion
      @favicon_contents = {"dc:title" => "Kuro5hin",}
```

Not in `Style/StringLiterals`.